RELEASING.md

```diff
@@ -0,0 +1,43 @@
+# How to release like real pro
+
+Releasing a new version of Connaisseur includes the following steps:
+
+- adding a new version tag
+- creating a changelog from commit messages
+- creating a PR from `develop` (new version) to `master` (current version)
+- pushing a new version of the Connaisseur image to Dockerhub
+- merging in the PR
```

Comment:
And pushing the tag with merge. If that was included in `adding a new version tag` I would reorder